definitions/features/katello_service.rb

```diff
@@ -0,0 +1,28 @@
+class Features::KatelloService < ForemanMaintain::Feature
+  metadata do
+    label :katello_service
+  end
+
+  def make_stop(spinner, args = '')
+    originally_running_services = running_services
+    spinner.update 'Stopping katello running services..'
+    execute!("katello-service stop #{args}")
```

Comment:
Oh, I'm missed there is the `except` word used. In that case, what about having `options` with possible keys `:only` and `:exclude`, but since we would have the list of currently running services, we would use only `--only` option for the actual `katello-service` and only in case we don't have an empty list.